code_samples/bigdecimal_sample.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// This sample code demonstrates BigDecimal usage.
+
+const { Client, BigDecimal } = require('..');
```

Comment:
```suggestion
const { Client, BigDecimal } = require('hazelcast-client');
```
so that the users would have an easier time copy-pasting and running the code in their locals